elastic/kibana
# [Index Management] Fix bug with long http requests coming from index actions

Closes https://github.com/elastic/kibana/issues/119704

## Summary
This PR patches all the server side actions of the index API in Kibana so that we take into account that there can be a max of 4096 bytes length in the parms sent to the ES API. The fix consists in splitting the list of indices being sent to ES into smaller chunks and waiting for all of them to resolve before sending the response back to the client.

I've also updated the toast messages not to include the list of index names in it, but instead to show how many indices were affected. Given that showing a list of affected indices isn't very helpful to the user and feels broken when selecting many indices (see screenshot).

#### How to test

Follow the instructions described [in this PR](https://github.com/elastic/kibana/issues/119704), make sure that staying the same page with pagination set to 100 does call the reload method and everything works as expected. Same for the bulk index actions.

<details>
<summary>Screenshots</summary>

![Screenshot 2023-11-22 at 13 40 08](https://github.com/elastic/kibana/assets/1191206/a80afc54-0d13-47f4-81e9-062b1fac66f7)


</details>
